src/main/java/bio/terra/common/CloudPlatformWrapper.java

```diff
@@ -56,11 +60,23 @@ void ensureValidRegion(CloudPlatform platform,
     public abstract List<? extends StorageResource<?, ?>>
         createStorageResourceValues(DatasetRequestModel datasetRequest);
 
+    public static Stream<? extends StorageResource<?, ?>>
+        createGoogleResourcesForAzure(DatasetRequestModel datasetRequestModel) {
```

Comment:
I'm wondering if `create` is the right name for these methods; maybe `get` would be better?
```suggestion
        getGoogleResourcesForAzure(DatasetRequestModel datasetRequestModel) {
```